packages/hash/api/src/seed-data/seed-pages.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { PageModel } from "../model";
+
+export type PageDefinition = {
+  title: string;
+  nestedPages: PageDefinition[];
+};
+
+export const seedPages = async (
+  pageDefinitions: PageDefinition[],
+  owningActorId: string,
+  sharedParams: {
+    graphApi: GraphApi;
+    logger: Logger;
+  },
+  parentPageModel: PageModel | null = null,
+) => {
+  const { graphApi } = sharedParams;
+
+  let prevIndex: string | undefined = undefined;
```

Comment:
Nit: variables are initialised to `undefined`
```suggestion
  let prevIndex: string | undefined;
```

Reply:
Addressed in a4632932ca9205874fd7142e2d0a106d1e15d0f5